app/recordtransfer/views/post_submission.py

```diff
@@ -122,7 +123,12 @@ def submission_group_bulk_csv_export(request: HttpRequest, uuid: str) -> HttpRes
         )
 
     if not related_submissions.exists():
-        raise Http404("No submissions found in this group")
+        messages.error(
+            request,
+            "This submission group has no submissions associated with it. Please add some submissions before attempting to export.",
```

Comment:
This message is a bit long, and needs translation. How about:

```suggestion
            _("This submission group has no submissions associated with it to export."),
```